packages/atomic/cypress/integration/breadbox/breadbox-assertions.ts

```diff
@@ -144,13 +144,6 @@ export function assertDeselectCheckboxFacet(
   });
 }
 
-export function assertDeselectCategoryFacet(index: number) {
```

Comment:
These checks were kind of duplicates, we already had assertions for validating that the facets were deselected correctly